src/styled.d.ts

```diff
@@ -67,6 +69,12 @@ declare module 'styled-components' {
       };
       size: {
         bodySmall: string;
+        label: string;
```

Comment:
Me either, but it is the same as the style guide for now. I am going to review it today and talk to Silvia about any names that don't really make sense.

Reply:
Can we maybe include a comment or something saying to review the name? I think it would be too easy for it to fall through the cracks if we don't leave ourselves a reminder

Reply:
I can, the reason I didn't is that it is next on my list. But that is fair, I'll update.